cypress/support/login.ts

```diff
@@ -0,0 +1,13 @@
+import { createClient } from '@supabase/supabase-js';
+
+export default function login() {
+  const appUrl: string = Cypress.env('supabaseAppUrl');
+  const publicAnonKey: string = Cypress.env('supabasePublicAnonKey');
+  const supabase = createClient(appUrl, publicAnonKey);
+
+  const email = 'testing@cast-iu.pages.dev';
```

Comment:
issue (non-blocking): might have been better not to hard-code this e-mail, but it's not a big deal

Reply:
I thought the same, but couldn't figure out where to store it instead. Having it in `.env` would be inconvenient since it will have to be added to all of our environments (including CI) when it's not really a secret. I guess I can just add it to `cypress.json`, since it's not even worth creating a separate `cypress.env.json` just for this one variable